src/pages/NftCollections/util.ts

```diff
@@ -0,0 +1,6 @@
+import { useSortParam } from '../../utils/hook'
+
+type NftSortByType = 'transactions' | 'holder' | 'minted'
+
+export const useNFTCollectionsSortParam = () =>
+  useSortParam<NftSortByType>(s => s === 'transactions' || s === 'holder' || s === 'minted', 'holder.asc')
```

Comment:
I would suggest using `['transactions', 'holder', 'minted'].includes(s)` instead of chained `or`. By that, `NftSortByType` can be defined as `typeof (['transactions', 'holder', 'minted'] as const)[number]`